helmfile/00-psp_rbac_np.yaml

```diff
@@ -0,0 +1,71 @@
+# Defult settings to use with helm.
```

Comment:
I'm guessing this file should be renamed `00-psp_rbac.yaml`. Now we have two of these

Reply:
> The only issue I see is that this is relying on a harbor instance managed by users. If the user deletes the proxy repo then this will stop working. So I think that I rather would like to see a fix in the opa policy to allow for these particular images rather than using harbor.

I didn't find a good way o add this exception and to make sure that no one can't take advantage of it. 
Another option would be to create and use an elastisys harbor repo (in a HA mode), we may also need it if we want to enable OPA for all namespaces in the future.

Reply:
I deleted the old 00-psp_rbac.yaml